src/nidmm/metadata/enums_addon.py

```diff
@@ -6,12 +6,14 @@
 # We also do not codegen enums associated with P2P or External Calibration since neither 
 # are supported in Python
 enums_codegen_method = {
-    'PowerlineFrequency': { "codegen_method": "no" }, # Enum metadata actually contains constants.
-    'CurrentSource': { "codegen_method": "no" }, # Enum metadata actually contains constants.
-    'InputResistance': { "codegen_method": "no" }, # Enum metadata actually contains constants.
 }
 
 enums_additional_enums = {
-    'DigitsResolution': {},  # Just delete. Otherwise we have to make it so that it will generate valid code
+    'DigitsResolution': {},  # Just delete this enum. Otherwise we have to make it so that it will generate valid code
```

Comment:
Enum metadata actually contains constants.

Reply:
That too. But we were deleting it even before because the values of it caused the code generator to break.

Reply:
Updated comments but still mention that invalid code would be generated by this enum